src/media/css/feed/shelves.styl

```diff
@@ -63,6 +63,47 @@ shelf_curve_color($color) {
     top: 80px;
 }
 
+.feed-shelf-landing-curve {
+    border-radius: 50% 50% 0 0;
+    height: 100px;
+    margin: 50px 0 0;
+}
+
+.feed-shelf-landing-inner {
+    margin-bottom: -45px;
+    padding-bottom: 30px;
+
+    .feed-landing-bg-image {
+        float: left;
+        margin-right: 30px;
+    }
+    > div {
+        max-width: $desktop-content;
+        position: absolute;
+        top: 40px;
+        width: 100%;
+    }
+    h1,
+    .feed-landing-description {
+        text-align: left;
+    }
+}
+.feed-shelf-info {
+    float: left;
```

Comment:
Yeah, flexbox is made for things like this. I've been using it quite a bit lately, holler if you'd like a sounding board for brainstorming.